### ISSUE TITLE
Website update

### ISSUE THREAD

**Issue by tabuna** [ratchetphp/socketo.me#33]
Hi, I can design a website to refresh it well. Are you interested in this?

**Comment by cboden** [comment]
Absolutely! I'm very, very interested! I've wanted to make a new one for a while but can never seem to find the time to do it.

**Comment by tabuna** [comment]
Oh, I can do it in the near future. Do you have preferences?

**Comment by cboden** [comment]
Nothing in particular. Use the colours in the Ratchet logo for the theme with a white background. Editing or adding new documents should be easy. 

What are your thoughts?

**Comment by tabuna** [comment]
Colors too complex, on a white background will look untidy.
The new documents will be using the markdown, okay?


I'm now going in the direction of this design:

![screencapture-localhost-8000-1508367629776](https://user-images.githubusercontent.com/5102591/31746769-be0ed1fc-b471-11e7-9425-5c89e5d439fd.png)
![screencapture-localhost-8000-docs-1508367492260](https://user-images.githubusercontent.com/5102591/31746770-be3209ce-b471-11e7-8096-3d8033b176a2.png)

**Comment by cboden** [comment]
> Colors too complex, on a white background will look untidy.

Ok. Design is not something I'm good at. I'll leave that to your expertise. 

> The new documents will be using the markdown.

Yup, that's awesome!

> I'm now going in the direction of this design

Looking good!  😀

**Comment by tabuna** [comment]
I think it's worth using a monochrome look

![logo](https://user-images.githubusercontent.com/5102591/31765417-f8d10c46-b4cc-11e7-92fd-f95266de656a.png)

**Comment by tabuna** [comment]
Do you have usage information?


I found only:
https://github.com/sulu/sulu
https://github.com/kraken-php/framework

**Comment by cboden** [comment]
Nope. I've only gone off [Packagist](https://packagist.org/packages/cboden/ratchet/dependents)

**Comment by tabuna** [comment]
may be
![dimanu](https://user-images.githubusercontent.com/5102591/32581314-8790eb80-c4fa-11e7-8fc4-dbbd471466c1.png)
